src/Chapter1.hs

```diff
@@ -620,8 +624,15 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 n = last1 n + last2 n
+    where
+        last1 x = mod (abs x) 10
+        last1 :: Int -> Int
+
+        last2 x = div (mod (abs x) 100) 10
+        last2 :: Int -> Int
```

Comment:
Great solution!

You could also reuse the `lastDigit` function in here, instead of `last1` 🙂